File: client/src/main/java/io/theam/client/commands/DeleteCustomerCommand.java
package io.theam.client.commands;

import com.github.rvesse.airline.annotations.Command;
import io.theam.client.service.RestClient;

@Command(name = "delete", description = "Delete a customer")
public class DeleteCustomerCommand extends BaseCommandWithId {

    @Override
    protected void doRun() {
        System.out.println("Removing customer with id: " + Long.toString(customerId));
        new RestClient(username, password).deleteCustomer(customerId);
    }
}

File: client/src/main/kotlin/io/theam/client/commands/DeleteCustomerCommand.kt
package io.theam.client.commands

import com.github.rvesse.airline.annotations.Command
import io.theam.client.service.RestClient

@Command(name = "delete", description = "Delete a customer")
class DeleteCustomerCommand : BaseCommandWithId() {

    override fun doRun() {
        println("Removing customer with id: " + java.lang.Long.toString(customerId!!))
        RestClient(username, password).deleteCustomer(customerId!!)
    }
}
